contracts/interfaces/IOnchainCredentialStatusResolver.sol

```diff
@@ -45,7 +45,7 @@ interface IOnchainCredentialStatusResolver {
      */
     struct CredentialStatus {
         IdentityStateRoots issuer;
-        Proof mtp;
+        MerkleTreeProof mtp;
```

Comment:
abi break, I'll think about it

Reply:
I'm talking about this repo
https://github.com/iden3/contracts-abi/blob/main/onchain-credential-status-resolver/go/abi/abi.go
which is used 
https://github.com/iden3/merkletree-proof/blob/main/eth/proof.go

Reply:
reverted and renamed Proof to ProofStorage in ZKPVerifierBase